pyaedt/modeler/object3dlayout.py

```diff
@@ -348,6 +348,224 @@ def enabled(self, status=True):
+        if "dt=1" in component_info:
+            return "FlipChip"
+        elif "dt=2" in component_info:
+            return "WireBond"
+        return None
+
+    @pyaedt_function_handler()
+    def set_die_type(
+        self,
+        die_type=1,
+        orientation=0,
+        height=0,
+        reference_offset=0,
+        auto_reference=True,
+        reference_x="0.1mm",
+        reference_y="0.1mm",
+    ):
+        """Set the die type
+
+        Parameters
+        ----------
+        die_type : int, optional
+            Die type. It can be 0 for None, 1 for FlipChip and 2 for WireBond.
+        orientation : int, optional
+            Die orientation. it can be 0 for Chip Top and 1 for Chip Bottom.
+        height : float, optional
+            Die height valid for port setup.
+        reference_offset : str, float, optional
+            Port reference offset. Default is 0.
+        auto_reference : bool, optional
+            Either if auto compute reference size. Default is `True`.
+        reference_x : str, float, optional
+            Reference x size. auto_reference as to be set to False to use this parameter.
+        reference_y : str, float, optional
+            Reference y size. auto_reference as to be set to False to use this parameter.
+
+        Returns
+        -------
+        bool
+        """
+        if self._part_type_id not in [0, 4, 5]:
+            return False
+        if auto_reference:
+            reference_x = "0"
+            reference_y = "0"
+        if self._part_type_id == 4:
+            prop_name = "ICProp:="
+        else:
+            prop_name = "IOProp:="
+        args = [
+            "NAME:Model Info",
+            [
+                "NAME:Model",
+                prop_name,
+                [
+                    "DieProp:=",
+                    ["dt:=", die_type, "do:=", orientation, "dh:=", str(height), "lid:=", -100],
+                    "PortProp:=",
+                    [
+                        "rh:=",
+                        str(reference_offset),
+                        "rsa:=",
+                        auto_reference,
+                        "rsx:=",
+                        reference_x,
+                        "rsy:=",
+                        reference_y,
+                    ],
+                ],
+                "CompType:=",
+                4,
+            ],
+        ]
+        return self.change_property(args)
+
+    @pyaedt_function_handler()
+    def set_solderball(
+        self, solderball_type="Cyl", diameter="0.1mm", mid_diameter="0.1mm", height="0.2mm", material="solder"
+    ):
+        """Set Solderball on the active component. It applies to Component of type Other, IC and IO.
+
+        Parameters
+        ----------
+        solderball_type : str, optional
+            Either one of "None", "Cyl" or "Sph" values.
+        diameter : str, optional
+            Ball diameter.
+        mid_diameter : str, optional
+            Ball diameter.
```

Comment:
```suggestion
            Ball diameter. The default is ``"0.1mm"``.
```

Reply:
The description in line 480 must differ from that in line 478. Perhaps it only needs to be changed to "Mid-ball diameter"?